modules/color-picker/react/lib/ColorInput.tsx

```diff
@@ -44,23 +49,37 @@ const CustomHexInput = styled(TextInput)<Pick<ColorInputProps, 'disabled' | 'gro
     },
   ({disabled}) => ({
     backgroundColor: disabled ? colors.soap200 : '',
+  }),
+  ({theme}) => ({
+    paddingLeft: theme.direction === ContentDirection.LTR ? '46px' : undefined,
+    paddingRight: theme.direction === ContentDirection.LTR ? undefined : 'calc(100% - 86px)',
```

Comment:
Not sure if you realize this, but this is getting flipped by rtl-css-js resulting in `padding-left`

Reply:
Yep, this is on purpose. Basically, I want `padding-left` to be `46px` on LTR but `calc(100% - 86px)` on RTL. If I do something like this:
`paddingLeft: theme.direction === C...n.LTR ? '46px' : 'calc(100% - 86px)',`

The result will be  `paddingRight : 'calc(100% - 86px)',`, which is the wrong side (because as you mentioned, it is getting flipped)

Reply:
I refactored this to use `/* @noflip */` that tells rtl-css-js to not flip.